src/main.rs

```diff
@@ -392,6 +392,14 @@ async fn token_fn(
                 // in /authorize as well
             )
             .await?;
+            tokio::spawn(async move {
+                if let Err(e) = tree.remove(data.clone()) {
+                    log::error!("{e}");
+                }
+                if let Err(e) = tree.remove(code.clone()) {
+                    log::error!("{e}");
+                }
```

Comment:
is this duplication to try two times? shouldn't be the one in the other then?

Reply:
I am removing two entries. One which its key is `data` and the other with key `code`

Reply:
ah i miss read sorry